### DISCUSSION TITLE
setup with nextjs

### DISCUSSION THREAD

**Discussion by mapsgeek** [jotaijs/jotai-devtools#77] [Q&A] [answered]
i have an issue setting up the devtools with nextjs for app directory and without
here's an example of what i do but nothing shows up in the tool:

https://codesandbox.io/p/sandbox/nextjs-with-jotai-forked-sqzxxt

**Comment by arjunvegda** [comment, marked as answer]
Support for Jotai `2.1.0` was added in `jotai-devtools@0.5.3`

Could you please try updating the `jotai-devtools` to the latest version and see if that fixes it?

**Reply by mapsgeek** [reply]
thank you it worked, i didn't notice i haven't added the latest version!
